src/server.ts

```diff
@@ -18,6 +18,7 @@ export const shutdown = () => {
     server.close();
 };
 const PORT: number | string = process.env.PORT || 3000;
+app.use(morgan('tiny'));
```

Comment:
Try to insert hard coded values into a separate constant/configuration file.

So you do not need to change multiple places when the value changes.

Reply:
@amr-imad